scripts/apps/search/directives/SearchResults.js

```diff
@@ -11,6 +11,12 @@ SearchResults.$inject = [
     'config'
 ];
 
+let HEX_REG_EXP = /[a-f0-9]{24}/;
```

Comment:
Since it's caps, maybe a `const` is better. Also, it doesn't seem to be needed outside the `isObjectId` function, so maybe it would be better in there?